src/Andegna/Calender.php

```diff
@@ -77,7 +77,7 @@ public static function ethiopianToJd($month = 1, $day = 1, $year = 2000)
         }
         return (1723856 + 365) +
         365 * ($year - 1) +
-        (int)($year / 4) +
```

Comment:
i think this part is making the test fail!
the `int` is used to cast (ignore the fraction in the calculation)
click [here](https://travis-ci.org/andegna/calender/builds/71723924) to see the test result